src/test/java/test/configuration/issue1035/MyFactory.java

```diff
@@ -0,0 +1,17 @@
+package test.configuration.issue1035;
+
+import java.util.Collections;
+import java.util.Set;
+import java.util.concurrent.ConcurrentHashMap;
+import org.testng.annotations.Factory;
+
+public class MyFactory {
+  public static Set<InvocationTracker> tracker = Collections.newSetFromMap(new ConcurrentHashMap<>());
```

Comment:
Could be final and use the constant naming convention (upper case)

Reply:
fixed